tools/clang/test/DXILValidation/compute_node_compatibility.hlsl

```diff
@@ -1,83 +1,37 @@
 // ==================================================================
 // Errors are expected for shaders with both "node" and "compute"
 // specified when:
-// - the launch type is not broadcasting
 // - a broadcasting node has an input record and/or output records
+// - the launch type is not broadcasting
```

Comment:
I don't think this comment is correct anymore.  This test is not about checking errors when both node and compute are specified, since we no longer support that scenario, right?  This is now a template used for validation of launch types with various input record types.  As such, the comment should probably be updated.